Artifacts/windows-slack/Artifactfile.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Slack",
+  "description": "Installs Slack using the Chocolatey package manager",
+  "tags": [
+    "Windows",
+    "Slack"
+  ],
+  "iconUri": "https://chocolatey.org/content/packageimages/slack.1.0.0.png",
+  "targetOsType": "Windows",
+   "parameters": {
+        "installUsername": {
```

Comment:
Makes sense. So what credentials are being supplied? The username and password of the VM user which is deployed with the VM, or are they Slack-specific? I think that ambiguity should be cleared up in the parameter descriptions.